src/AutoMapper/MappingExpressionFeatureCollectionExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace AutoMapper
+{
+    public static class MappingExpressionFeatureCollectionExtensions
```

Comment:
I don't this this is needed.

Reply:
No, correct. It was some code that I need before I added the `Get`/`Set` on the collection itself. I will remove.